### PR TITLE
Change Site Domain to Frontend URL

### PR DESCRIPTION
**Pull request by seporterfield**
These changes introduce
- the ability to change the base URL in activation emails
- a convenience script to do so
- new changes to .template-env

The site domain is stored only in the **database** and is not decided by the **application**.

### CODE REVIEW

File: backend/scripts/update_site_domain.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+# running on.
+# This is the URL sent in activation emails.
+#
+# To run this script, pipe it into the Django shell.
+# python manage.py shell < [this file]
+
+from django.conf import settings
+from django.contrib.sites.models import Site
+
+site = Site.objects.get(id=1)
```
Comment (Monstarrrr) [inline backend/scripts/update_site_domain.py:11]: Why not do ``get(id=SITE_ID)`` ?

File: backend/rebutify/settings.py [hunk 2]
```diff
@@ -38,6 +38,8 @@
     else ["localhost", "127.0.0.1"]
 )
 
+SITE_URL = os.getenv("FRONTEND_SITE_URL", "localhost:3000")
```
[the remark this answers is not in the document]
Comment (seporterfield) [reply backend/rebutify/settings.py:41]: Is it not self documenting?

Do the [docs](https://docs.python.org/3/library/os.html#os.getenv) for os.getenv not suffice?

Or is this a suggestion about the variable name?

[the remark this answers is not in the document]
Comment (Monstarrrr) [reply backend/rebutify/settings.py:41]: I want as many people as possible being able to navigate and understand the code without having to pull out the docs frequently, it is more welcoming and inclusive which is important if we want anyone to consider contributing.  

For that I would comment framework functions that are not frequently used, unless they are self-explanatory from their name.  

This is something I will document as part of contributing guidelines and open a decision for, and we can discuss it in its decision PR, and until then it's up to you

[the remark this answers is not in the document]
Comment (seporterfield) [reply backend/rebutify/settings.py:41]: Do we want anyone to consider contributing?
People who need a comment for standard library functions like print() and os.getenv()?

Or people who can make useful contributions?

[the remark this answers is not in the document]
Comment (seporterfield) [reply backend/rebutify/settings.py:41]: > I'd add a comment   

You're right that this setting is missing a comment explaining what it is though, I'll get on that

[the remark this answers is not in the document]
Comment (Monstarrrr) [reply backend/rebutify/settings.py:41]: > Anyone?
> 
> Pulling up the docs is part of programming, and standard library functions like os.getenv() come up so often that you more often internalize what it does.
> 
> Also, contributors would be looking at the code in their text editor/IDE. Intellisense (mouse hover over the function name) can tell them exactly what these functions do. They are already documented by the python community, and we don't need to double the work.
> 
> Do we want anyone to consider contributing?
> 
> People who need a comment for standard library functions like print() and os.getenv()?
> 
> Or people who can make useful contributions?

Yes if it's so common as you say it's unecessary for that one

**Pull request merged**